### PR TITLE
Standardized Error Messages

### PR DESCRIPTION
**Pull request by MM-coder**
Implemented the `discordError()` class in the `helpers.py` file.

### CODE REVIEW

File: commands/xp.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import utils.colours as colour
 import utils.values as value
 import utils.database as sql
-from utils.helpers import discordError
+from utils.helpers import discordError, cinfo, add_exp
```
Comment (MattIBM) [inline commands/xp.py:7]: Please don't do this, we import the whole of helpers for a reason. Also, cinfo does not exist. Please just import utils.helper and not worry about what methods to include.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+__pycache__/
```
Comment (MattIBM) [inline .gitignore:1]: This is out of scope of the PR,

File: utils/helpers.py [hunk 3]
```diff
@@ -181,3 +181,14 @@ def time_phaser(seconds):
     if s > 0:
         output = output + str(int(round(s, 0))) + " Seconds "
     return output
+
+
+async def discordError(errorMessage: str, ctx):
```
Comment (MattIBM) [inline utils/helpers.py:186]: Should be discord_error()

File: commands/puppy.py [hunk 4]
```diff
@@ -14,7 +15,7 @@ def __init__(self, client):
     @commands.command()
     async def puppy(self, ctx):
         log.debug("%s issued server command %s" % (str(ctx.message.author), str(ctx.message.content)))
-        imgUrl = random.choice(["https://media3.giphy.com/media/HKl5QYQF5aOdO/giphy.gif", "http://giphygifs.s3.amazonaws.com/media/bmrxNoGqGNMAM/giphy.gif", "https://media0.giphy.com/media/xT0xeuOy2Fcl9vDGiA/giphy.gif", "https://media3.giphy.com/media/73h3LBWraONTW/giphy.gif"])
+        imgUrl = requests.get('https://dog.ceo/api/breeds/image/random').json()['message']
         em = discord.Embed(title="Puppy", image=imgUrl, colour=colour.secondary)
         em.set_image(url=imgUrl)
         await ctx.send(embed=em)
```
Comment (MattIBM) [inline commands/puppy.py:21]: Out of scope for this PR.

File: commands/xp.py [hunk 5]
```diff
@@ -20,15 +20,15 @@ async def xp(self, ctx, targetMember : discord.Member = None):
             if args[2].upper() == "GIVE":
                 amount = args[3]
                 if int(amount) < 0:
-                    discordError("Cannot give negative XP!", ctx)
+                    await discord_error("Cannot give negative XP!", ctx)
                 else:
                     add_exp(member.id, int(amount))
                     em = discord.Embed(description=":ok_hand: Successfully given **%s** **%s EXP**!" % (member.name, amount), color=colour.primary)
                     em.set_author(name="Operation Complete")
```
Comment (MattIBM) [inline commands/xp.py:27]: Could this perhaps say "Success"? I'm not a big fan of "Operation Complete"

Comment (MM-coder) [reply commands/xp.py:27]: On it

File: commands/kitten.py [hunk 6]
```diff
@@ -15,7 +14,7 @@ def __init__(self, client):
     @commands.command()
     async def kitten(self, ctx):
         log.debug("%s issued server command %s" % (str(ctx.message.author), str(ctx.message.content)))
-        imgUrl = requests.get('https://api.thecatapi.com/v1/images/search').json()[0]['url']
+        imgUrl = random.choice(["https://cdn.discordapp.com/avatars/283388594608013312/ddc6f6ae51689a382eecc100103e1b79.png?size=1024", "https://media.tenor.com/images/fe9cdda998e9d121f318c2d938c9c6a2/tenor.gif", "https://media.giphy.com/media/kvrvnB158J4fm/giphy.gif", "https://media.giphy.com/media/r1OyZ5NfRJigg/giphy.gif", "https://media.giphy.com/media/4rep3f9ih9u12/giphy.gif"])
```
Comment (MattIBM) [inline commands/kitten.py:17]: Conflicts with what is already delivered.

**Comment by MM-coder** [comment]
Ready for review and merge

**Comment by MM-coder** [comment]
@MattIBM The check appears to be broken

**Comment by MattIBM** [comment]
Should be fixed now. Try making some random commit

**Comment by MattIBM** [comment]
Check broke, re-create PR.
